features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/base64/Base64LRUCache.kt

```diff
@@ -16,14 +16,18 @@ import android.util.LruCache
 import androidx.annotation.VisibleForTesting
 import com.datadog.android.sessionreplay.internal.utils.CacheUtils
 
-internal object Base64LRUCache : Cache<Drawable, String>, ComponentCallbacks2 {
-    @Suppress("MagicNumber")
-    private val MAX_CACHE_MEMORY_SIZE_BYTES = 4 * 1024 * 1024 // 4MB
-
-    private var cache: LruCache<String, ByteArray> = object :
-        LruCache<String, ByteArray>(MAX_CACHE_MEMORY_SIZE_BYTES) {
-        override fun sizeOf(key: String?, value: ByteArray): Int {
-            return value.size
+internal class Base64LRUCache(
```

Comment:
it doesn't make sense to have a `BaseLruCache` without a backup `cache` so the `cache` should not be optional here.